checker/src/javax/annotation/concurrent/GuardedBy.java

```diff
@@ -41,19 +44,23 @@
 // to the Lock Checker's @GuardedBy annotation) or on a method (in which case
 // it is a declaration annotation corresponding to the Lock Checker's @Holding
 // annotation).
+// It is preferred to use these Checker Framework annotations instead:
+//  org.checkerframework.checker.lock.qual.GuardedBy
+//  org.checkerframework.checker.lock.qual.Holding
```

Comment:
People do use it, so if we want to support it, TYPE_USE is necessary. Mike suggested that we may want to drop support for the javax and jcip GuardedBy annotations since they are poorly defined. What do you think - should we drop support for them?